biojava-core/src/main/java/org/biojava/nbio/core/sequence/io/ABITrace.java

```diff
@@ -0,0 +1,568 @@
+/*
+ *                    BioJava development code
+ *
+ * This code may be freely distributed and modified under the
+ * terms of the GNU Lesser General Public Licence.  This should
+ * be distributed with the code.  If you do not have a copy,
+ * see:
+ *
+ *      http://www.gnu.org/copyleft/lesser.html
+ *
+ * Copyright for this code is held jointly by the individual
+ * authors.  These should be listed in @author doc comments.
+ *
+ * For more information on the BioJava project and its aims,
+ * or to join the biojava-l mailing list, visit the home page
+ * at:
+ *
+ *      http://www.biojava.org/
+ *
+ * Created on 05-04-2018
+ */
+
+package org.biojava.nbio.core.sequence.io;
+
+import java.io.IOException;
+import java.io.File;
+import java.io.BufferedInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.FileInputStream;
+import java.io.DataInputStream;
+import java.io.ByteArrayInputStream;
+import java.awt.Color;
+import java.awt.Graphics2D;
+import java.awt.image.BufferedImage;
+import java.net.URL;
+import java.io.InputStream;
+
+import org.biojava.nbio.core.sequence.compound.ABITracerCompoundSet;
+import org.biojava.nbio.core.sequence.compound.DNACompoundSet;
+import org.biojava.nbio.core.sequence.compound.NucleotideCompound;
+import org.biojava.nbio.core.sequence.template.AbstractSequence;
+import org.biojava.nbio.core.exceptions.CompoundNotFoundException;
+
+/**
+ * Title: ABITrace<br><br>
+ * ABITrace is a class for managing ABI file information,
+ * it is capable of opening an ABI file and storing
+ * the most important fields, which can be recalled as simple java types. It can also return
+ * an image corresponding to the trace.
+ * It has three constructors with input types <code>File, URL, and byte[]</code>.<br><br>
+ * ABI files contain two sets of basecall and sequence data, one that was originally
+ * created programatically and the other, which is an editable copy. This version of this object
+ * only references the original unedited data.<br>
+ */
+public class ABITrace {
+
+    //the next three lines are the important persistent data
+    private String sequence;
+    private int A[], G[], C[], T[], Basecalls[], Qcalls[];
+    private int TraceLength, SeqLength;
```

Comment:
Variables should start with lower case as per normal java conventions.

I know this was like that in legacy project, but it'd be a great opportunity to fix the style while porting it :)